src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -359,6 +360,10 @@ impl<'a> Linker for GccLinker<'a> {
         self.cmd.arg("__llvm_profile_runtime");
     }
 
+    fn control_flow_guard(&mut self) {
+        // Windows Control Flow Guard (/guard:cf) is not yet supported by this linker.
```

Comment:
I think it would be better to report an error instead of silently ignoring this

Reply:
Thanks for the suggestion! Would it be sufficient to give a warning in each of these cases or should this be a fatal error?

Reply:
Not sure, I'll let the compiler team decide

Reply:
Thanks - is there something I should do to ask for further review?

Reply:
No, it just might take a while until the reviewer can get to it